package.json

```
@@ -57,8 +57,8 @@
     "test:sauce:desktop": "./script/run-nightwatch.sh --sauce -e chrome,firefox,ie,edge",
     "test:sauce:mobile": "./script/run-nightwatch.sh --sauce -e ios,android",
     "test:unit": "node ./script/run-unit-test.js",
-    "test:unit:form-system": "BABEL_ENV=test npx mocha --opts src/platform/testing/unit/mocha.opts --recursive 'src/platform/forms-system/test/js/**/*.unit.spec.js?(x)' src/platform/testing/unit/helper.js '$@'",
     "test:unit:forms": "BABEL_ENV=test npx mocha --opts src/platform/testing/unit/mocha.opts --recursive 'src/platform/forms/tests/**/*.unit.spec.js?(x)' src/platform/testing/unit/helper.js '$@'",
+    "test:unit:form-system": "BABEL_ENV=test npx mocha --opts src/platform/testing/unit/mocha.opts --recursive 'src/platform/forms-system/test/js/**/*.unit.spec.js?(x)' src/platform/testing/unit/helper.js '$@'",
```

```suggestion
    "test:unit:forms-library": "BABEL_ENV=test npx mocha --opts src/platform/testing/unit/mocha.opts --recursive 'src/platform/forms-system/test/js/**/*.unit.spec.js?(x)' src/platform/testing/unit/helper.js '$@'",
```
I know the directory is called `forms-system`, but our word list says [forms library](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/documentation/style-guide/word-list.md#forms-library). :man_shrugging: